cypress/e2e/api.spec.cy.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+import { onLandingPage } from "../support/pageObject/landingPage";
+
+describe("API intercept", () => {
+  beforeEach("open app", () => {
+    cy.log("opening web page");
+    onLandingPage.openBasePage();
+  });
+
+  it("Tags request", () => {
+    cy.intercept({
```

Comment:
can we make the same as for POM factory and move cy.intercept as a custom methods? But overall this is exactly what I asked for :)

Reply:
not sure if this is what was requested. Let's discuss tomorrow during the call